bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Path.java

```diff
@@ -695,4 +699,20 @@ public String toString() {
 	return "Path {" + handle + "}";
 }
 
+/**
+ * Gets the handle for the path at the specified zoom level
+ *
```

Comment:
The comment is missing the `@noreference`, `@param` and  `@return ` values and the please to not use part

Reply:
Done